### PR TITLE
Dependent chart test

### PR DESCRIPTION
**Pull request by scottrigby**
Initially this should fail

### CODE REVIEW

**Comment by scottrigby** [comment]
Increased ct helm install timeout. Previously Redis master PVC did not bind in time. For not just added the same timeout as https://github.com/helm/charts/blob/master/test/ct.yaml. Let's see if that resolves it.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -17,6 +17,21 @@ jobs:
...
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      # See https://github.com/helm/chart-releaser-action/issues/6
+      # For now, install the same version of Tiller as the chart-testing Helm
+      # client version: v2.15.2.
+      - name: Install Helm
+        run: |
+          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get
+          chmod 700 get_helm.sh
+          DESIRED_VERSION=v2.15.2 ./get_helm.sh
```
Comment (unguiculus) [inline .github/workflows/release.yaml]: v2.16.1?

Comment (unguiculus) [reply .github/workflows/release.yaml]: What's the env for? Doesn't seem to be used.

**Review by unguiculus — DISMISSED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (scottrigby) [reply .github/workflows/release.yaml]: Perhaps the minor version doesn't matter anymore. Initially I did this for compatibility with our current helm/chart-testing-action helm version, but we could remove that here.

The `DESIRED_VERSION` is from the Helm 2 get script: https://github.com/helm/helm/blob/02ad2b118731d012ea248e3ede800aa28fa71301/scripts/get#L79-L94

Comment (scottrigby) [reply .github/workflows/release.yaml]: I removed that, so it should just install the most recent version of Helm 2 now

File: .github/workflows/lint-test.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
         uses: actions/checkout@v1
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.0.0-alpha.2
+        uses: helm/kind-action@master
         with:
           installLocalPathProvisioner: true
```
Comment (davidkarlsen) [inline .github/workflows/lint-test.yaml]:
```suggestion
          install_local_path_provisioner: true
```

Comment (scottrigby) [reply .github/workflows/lint-test.yaml]: Ah too late, thank you yes I completely forgot to update the input key after https://github.com/helm/kind-action/pull/10 💯

**Comment by scottrigby** [comment]
OK it works 👍  let's release a new version of helm/kind-action and update the version here before merging this PR

**Pull request merged**